src/CityDisplay.js

```diff
@@ -0,0 +1,89 @@
+import './App.css';
+import React from 'react';
+import axios from 'axios';
+
+class CityDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCityData: false,
+      cityData: [],
+      userSearch: '',
+      errMsg: '',
+      errRender: false,
+
+    }
+  }
+
+  handleExplore = async () => {
+
+    try {
+      let cityResults = await axios.get('https://us1.locationiq.com/v1/search.php')
```

Comment:
- I see that we have this method which is grabbing city data as well as another method below that is grabbing city data
- Your handleExplore is really responsible for handling some sort of event, in this case, when the user types in a city name, we will set the state using whatever the e.target.value is 👍